tests/test_executor.py

```diff
@@ -289,11 +289,6 @@ def test_set_operations(self):
                 ["a"],
                 [(1,), (2,), (3,)],
             ),
-            (
-                "SELECT 1 AS a UNION SELECT 2 AS a UNION SELECT 3 AS a",
-                ["a"],
-                [(1,), (2,), (3,)],
-            ),
```

Comment:
Why was this removed?

Reply:
Its a duplicate of the test directly above

Reply:
oh missed that, nice catch 👍